src/_scss/pages/award/awardFinancialAssistance.scss

```diff
@@ -23,6 +23,146 @@
                     margin: 0;
                 }
             }
+              width: 100%;
+              .visualization-tooltip {
+                @import "components/visualizations/tooltip/_tooltip";
+                position: fixed;
+                top: 0;
+                left: 0;
+            
+                .tooltip {
+                  position: relative;
+                  max-width: rem(500);
+                  .tooltip-title {
+                    background-color: #eeeeee;
+                  }
+                  .tooltip-subtitle {
+                    text-align: center;
+                    background-color: #f8f8f8;
+                    padding: $global-pad ($global-pad * 2);
+                  }
+                  .disclaimer-total {
+                      margin-bottom: rem(15);
+                      padding: rem(0) rem(15);
+                      font-size: rem(12);
+                      font-style: italic;
+                      text-align: center;
+                  }
+                }
+              }
+              @import '../search/results/table/tableMessages';
+              .results-table-message-container {
+                position: relative;
+                background-color: rgb(246, 246, 246);
+                .results-table-loading {
+                  margin-bottom: rem(100);
+                }
+                .no-results-container {
+                  text-align: center;
+                  margin-top: rem(100);
+                  margin-bottom: rem(100);
+                  .no-results-title {
+                    h4 {
+                      margin-bottom: 0;
+                      vertical-align: bottom;
+                    }
+                  }
+                  .no-results-message {
+                    p {
+                      margin-top: 0;
+                      vertical-align: top;
+                    }
+                  }
+                }
+                .loading-animation {
+                    @import "../search/results/screens/loading";
+                }
+                .results-table-error {
+                  .icon {
+                    width: rem(36);
+                    height: rem(36);
+                    svg {
+                      width: rem(36);
+                      height: rem(36);
+                    }
+                  }
+                  .title {
+                    font-size: rem(20);
+                    line-height: rem(45);
+                  }
+                  .description {
+                    font-size: rem(16);
+                    line-height: rem(10);
+                  }
+                }
+            }
+              .cfda-section__viz {
```

Comment:
Could this block be moved to its own file?

Reply:
yeah